code/modules/clothing/under/miscellaneous.dm

```diff
@@ -254,6 +254,78 @@ obj/item/clothing/under/genericr
 	style = STYLE_HIGH
 	price_tag = 300
 
+//stylish vendor clothes
+/obj/item/clothing/under/style
+	name = "ivory white outfit"
+	desc = "White suit, white pants and a white tie. Very stylish."
+	icon_state = "suit_white"
+	item_state = "suit_white"
+	style = STYLE_HIGH
+	price_tag = 450
+
+/obj/item/clothing/under/style/red
+	name = "blood red outfit"
+	desc = "Red suit, red pants and a red tie. Very stylish."
+	icon_state = "suit_red"
+	item_state = "suit_red"
+
+/obj/item/clothing/under/style/green
+	name = "questionable outfit"//Riddler reference, you probably knew that
+	desc = "Green suit, green pants and a green tie. Quite a riddle to solve."
+	icon_state = "riddler"
+	item_state = "riddler"
+
+/obj/item/clothing/under/style/grey
+	name = "ash grey outfit"
+	desc = "Grey suit, grey pants and a grey tie. Very stylish."
+	icon_state = "suit_grey"
+	item_state = "suit_grey"
+
+/obj/item/clothing/under/style/black
+	name = "charcoal black outfit"
+	desc = "Black suit, black pants and a black tie. Very stylish."
+	icon_state = "suit_black"
+	item_state = "suit_black"
+
+/obj/item/clothing/under/style/dress
+	name = "black and purple dress"
+	desc = "A black and purple dress, quite stylish"
+	icon_state = "black_purple"
+	item_state = "black_purple"
+
+/obj/item/clothing/under/style/dress/white
+	name = "white and yellow dress"
+	desc = "A white and yellow dress, quite stylish"
+	icon_state = "white_yellow"
+	item_state = "white_yellow"
+
+/obj/item/clothing/under/style/johnny
+	name = "rockerboy outfit"//Cyberpunk 2077, Johnny Silverhand's outfit
+	desc = "A padded vest with brown leather pants. Never stop fightin'"
+	icon_state = "johnny"
+	item_state = "johnny"
+	price_tag = 600
+	armor = list(
+		melee = 10,
+		bullet = 5,
+		energy = 5,
+		bomb = 0,
+		bio = 0,
+		rad = 0
+	)
+
+/obj/item/clothing/under/style/johnny/raider
+	name = "leather outfit"
+	desc = "A rather generic leather outfit. Smells like waste"
+	icon_state = "raider"
+	item_state = "raider"
+
+/obj/item/clothing/under/style/johnny/helltaker
```

Comment:
why is this a child of johnny

Reply:
my bad, must have accidentaly copied it

Reply:
done, now its like the other normal uniforms